integration-auth/src/main/java/org/symphonyoss/integration/authentication/jwt/JwtAuthentication.java

```diff
@@ -57,10 +84,20 @@
   private IntegrationBridge integrationBridge;
 
   @Autowired
+  private IntegrationProperties properties;
+
+  @Autowired
   private IntegrationHttpApiClient integrationHttpApiClient;
 
+  @Autowired
+  private RsaKeyUtils rsaKeyUtils;
+
   private IntegrationAuthApiClient apiClient;
 
+  private PublicKey podPublicSignatureVerifier;
+
+  private Date podPublicSignatureVerifierExpirationDate;
```

Comment:
Only using because I have the before() method to work with. The problem with Google Guava is that I have to have a key to find the cached resource, in this case I don't have it.